packages/orbit-components/src/OrbitProvider/types.d.ts

```diff
@@ -8,4 +8,5 @@ import type { Theme } from "../defaultTheme";
 export interface Props {
   readonly theme: Theme;
   readonly children: React.ReactNode;
+  readonly useId: typeof React.useId;
```

Comment:
Is `react-uid` compliant with this type?

Reply:
yeah, I'll simplify it to `() => string`.